src/rootPages/Designer/properties/process/ABProcessTriggerTimer.js

```diff
@@ -0,0 +1,255 @@
+/*
+ * UIProcessTriggerTimer
+ *
+ * Display the form for entering the properties for a new
+ * Timer Trigger.
+ *
+ * @return {ClassUI} The Class Definition for this UI widget.
+ */
+import UI_Class from "../../ui_class";
+
+export default function (AB) {
+   const UIClass = UI_Class(AB);
+   const L = UIClass.L();
+   const uiConfig = AB.Config.uiSettings();
+
+   class UIProcessTriggerTimer extends UIClass {
+      constructor() {
+         super("properties_process_trigger_lifecycle", {
+            name: "",
+            repeatEvery: "",
+            repeatTime: "",
+            repeatOnPanel: "",
+            repeatDaily: "",
+            repeatWeekly: "",
+            repeatMonthly: "",
+            isEnabled: "",
+            triggerKey: "",
+         });
+      }
+
+      static get key() {
+         return "TimerStartEvent";
+      }
+      // {string}
+      // This should match the ABProcessTriggerTimerCore.defaults().key value.
+
+      ui() {
+         let ids = this.ids;
+
+         let dayOptions = [];
+         for (let day = 1; day <= 31; day++) {
```

Comment:
What happens if we select `31`? If the month doesn't have 31 days it wouldn't run?
Should we limit the options, or maybe just add help text warning that it won't run every month?

Reply:
good question.

That is the code we were using in our previous version.  So the functionality is what we had before. 

The `node-cron` package does indicate that is a valid input into the field.  But a quick google on some cron explanations indicate that you would only choose `31` if you want it to run on the 31st of a month, which means it wouldn't run on months with < 31 days.

There are a few discussions about how to schedule `end of the month`, and most of them require 3 cron entries to accomplish.  You could however schedule the `1st of the month at 12:00 am` to simulate end of the month.

So for the moment, I think we would leave it as is since it is a valid specification.  But us builders need to know that it is very precise and only means that it will run on months with 31 days.